### ISSUE TITLE
Display of tabs from top  to bottom after exiting incognito

### ISSUE THREAD

**Issue by basimaslamts** [bromite/bromite#806]
### Is your feature request related to privacy?
No

### Is there a patch available for this feature somewhere?
No

### Describe the solution you would like
Exiting from incognito should direct the display of the tab from bottom to top

Reproduction
---------------------
I have about 100 tabs open. And the last few tabs will be what I will be using. When exiting from incognito, The first tabs are displayed and everytime I have to scroll down to see the tab I'm actually using.

I have enabled tab groups and other associated flags such as UI improvements

### Describe alternatives you have considered
Nil

The steps to repro should be as follows:

Have over a 100 tabs.
1.Open grid tab switcher with tab groups and 2.all other tab related flags in Chrome 87 enabled.
3. Switch to incognito mode
4. Switch back to normal mode

Expected behaviour:
GTS would be at the bottom, where my active tab is.

Actual behaviour:
GTS has scrolled to the top. Pressing back still takes me to my active tab at the bottom, however. I can then tap on the tab switcher button and it'll have scrolled back down again.

**Comment by csagan5** [comment]
@basimaslamts thanks for your submission, however this is not related to privacy and we close them as a rule.

@uazo would it be easy to have the tab view pre-scrolled to bottom? Otherwise please don't spend time on this issue; it is related to UI.

**Comment by Poopooracoocoo** [comment]
this is a bug with the tab grid switcher (GTS). you should try getting the chromium devs to fix it. I have yet to get them to fix a single issue so far though. they're really bad. they work on stuff like site info v2 instead of actual bugs or real features like what bromite has or like bookmark all tabs. smh. Google sucks.

**Comment by basimaslamts** [comment]
reported this bug in https://bugs.chromium.org/
under 
https://crbug.com/1151944

**Comment by Poopooracoocoo** [comment]
I've reported it too, submitted feedback, still nothing. They couldn't even triage mine properly.

**Comment by csagan5** [comment]
@basimaslamts they are requesting user feedback there on the issue tracker.

**Comment by Poopooracoocoo** [comment]
@basimaslamts the way you phrased the issue would be very confusing to the triagers whose english is not their first language and who often have a hard time understanding what people say. They even confused _switch to tab from the omnibox_ and _search tabs from the tab switcher_! You could add this as a comment if you'd like :)

The steps to repro should be as follows:

1. Have over a 100 tabs.
2. Open grid tab switcher with tab groups and all other tab related flags in Chrome 87 enabled.
3. Switch to incognito mode
4. Switch back to normal mode

Expected behaviour: 
GTS would be at the bottom, where my active tab is. 

Actual behaviour:
GTS has scrolled to the top. Pressing back still takes me to my active tab at the bottom, however. I can then tap on the tab switcher button and it'll have scrolled back down again.

An alternative STR:
1. Have over 100 tabs.
2. Open grid tab switcher with tab groups and all other tab related flags in Chrome 87 enabled.
3. Press on menu and tap on group tabs

Same expected behaviour and same actual behaviour.



i still get sad about there not being bookmark all tabs. i just remembered the times when i lost the tabs i had in incognito... 😢

**Comment by basimaslamts** [comment]
What does it mean
![IMG_20201126_104949](https://user-images.githubusercontent.com/57472154/100311182-d8567f00-2fb7-11eb-94ea-f472645b6235.jpg)

**Comment by Poopooracoocoo** [comment]
That just means that the bot sent a message to the triager of the issue that you gave them more information.

**Comment by basimaslamts** [comment]
![IMG_20201130_191940](https://user-image@Poopooracoocoo ithubusercontent.com/57472154/100617979-3630f100-3341-11eb-9a3f-8ab7394d2956.jpg)

@Poopooracoocoo thank you for ur support 😀

**Comment by basimaslamts** [comment]
@Poopooracoocoo thank you for ur help😀
![IMG_20201130_191940](https://user-images.githubusercontent.com/57472154/100618133-6d070700-3341-11eb-88ff-bb6feac080f2.jpg)

**Comment by basimaslamts** [comment]
Similar complaint has already raised under ..

https://bugs.chromium.org/p/chromium/issues/detail?id=1131341

**Comment by Poopooracoocoo** [comment]
@basimaslamts What were you confused about by the way?

**Comment by basimaslamts** [comment]
Partially resolved

**Comment by csagan5** [comment]
> Partially resolved

How has it been partially resolved?

**Comment by basimaslamts** [comment]
though not confirmed (but asked to check in the new chromium version)  in this issue (not closed yet)  https://bugs.chromium.org/p/chromium/issues/detail?id=1131341  ,
I couldn't replicate the issue in new version

**Comment by basimaslamts** [comment]
due to some  technical misunderstanding I couldn't reproduce the issue. apologies

**Comment by basimaslamts** [comment]
I think the issue has solved with new update. So closing issue. Thank you

**Comment by Poopooracoocoo** [comment]
This has not been resolved yet. We're still waiting on Google to fix this, @basimaslamts 

Have you perhaps enabled some flags? I've enabled and disabled flags and didn't see a fix for this.

**Comment by basimaslamts** [comment]
I use bromite 89.0.4389.92
I've enabled Tab groups, Tab continuation, top groups UI improvements

I've tested multiple Times in my Redmi 7, Miui Android 10. 

Issue is resolved to me.

**Comment by Poopooracoocoo** [comment]
@csagan5 since this can be reproduced in chromium, did you want to close this issue?

**Comment by basimaslamts** [comment]
@Poopooracoocoo whats your status with bromite?

**Comment by Poopooracoocoo** [comment]
@basimaslamts not affiliated with bromite at all. i just know that the dev likes to keep few issues and is quite strict

**Comment by csagan5** [comment]
The upstream issue was merged into https://bugs.chromium.org/p/chromium/issues/detail?id=1131341 which was then closed; I think we can do a fix in Bromite for this.

**Comment by basimaslamts** [comment]
@csagan5 are you Currently able to replicate this issue in ur bromite?

**Comment by basimaslamts** [comment]
I think the issue is resolved then

**Comment by csagan5** [comment]
It is no more reproducible.
